clang/test/SemaCXX/constexpr-vectors.cpp

```diff
@@ -1,10 +1,6 @@
-// RUN: %clang_cc1 -std=c++14 -Wno-unused-value %s -disable-llvm-passes -triple x86_64-linux-gnu -emit-llvm -o - | FileCheck %s
+// RUN: %clang_cc1 %s -Wno-uninitialized -std=c++17 -fsyntax-only -verify
```

Comment:
Why did the c++ standard change?

Reply:
To use C++ extension for static_assert, I can back to c++14 and add empty message if that needed

Reply:
Yes please, so we can keep the test as it was.

Reply:
Sure i will, thanks

Reply:
@tbaederr Done